packages/cockroach/extra/cockroachdb-change-config.py

```diff
@@ -107,9 +113,6 @@ def main() -> None:
 
     set_num_replicas(my_internal_ip, master_node_count)
 
-    # We are running CockroachDB v2.0.x so pass '2.0'.
-    set_cluster_version(my_internal_ip, '2.0')
```

Comment:
If we no longer need this, should we be explictly unsetting this for 2.1?

Reply:
No, it will automatically be set to `2.1`